woodwork/tests/schema/test_column_schema.py

```diff
@@ -410,3 +410,9 @@ def test_schema_repr():
             "<ColumnSchema (Semantic Tags = ['category', 'foreign_key'])>")
     assert (repr(ColumnSchema()) ==
             "<ColumnSchema>")
+
+
+def test_ordinal_without_init():
```

Comment:
I'm I'm understanding correctly, this should also be possible in the Table Schema--can we add a similar test there as well?

Reply:
Yep - added!